scripts/maintenance/import_projects_as_template.py

```diff
@@ -100,7 +114,7 @@ async def import_project_as_template(
     password: SecretStr,
     project_file: Path,
     project_name: str,
-    share_with_gid: int,
+    share_with_gid: int,  # TODO: not used!?
```

Comment:
hmm yes this went forgotten. this is in case you want to share the template with another group than everyone...